resources/python.rb

```diff
@@ -1,66 +1,59 @@
-#
-# Cookbook:: pyenv
-# Resource:: python
-#
-# Author:: Shane da Silva
-# Author:: Darwin D. Wu <darwinwu67@gmail.com>
-#
-# Copyright:: 2014-2017, Shane da Silva
```

Comment:
These really need to stay.

Reply:
Apart from this code was hijacked fro mthe ruby_rbenv cookbook, so I'm not really sure it's correct.